2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -133,13 +139,22 @@ class CollectorService(
         `Access-Control-Allow-Headers`("Content-Type")
       ))
 
+  override def flashCrossDomainPolicy: HttpResponse =
```

Comment:
I thought we don't use `override` unless we're overriding behavior? In `Service` this method is empty.

Reply:
I tend to always write `override`, this way you know this method was first defined elsewhere (here in the trait).

Here it's not the case but if this was still abstract you have the added benefit of signature checking between this and the overriden method instead of defining a new method.